### PR TITLE
l10n_no: change report format to rf-0002

### PR DESCRIPTION
**Pull request by nla-odoo**
Task : https://www.odoo.com/web#id=1969011&action=327&model=project.task&view_type=form&menu_id=4720

Pad: https://pad.odoo.com/p/r.27eb04e13e8e3e3e624b0d80e13acb2a

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_no/data/account_tax_report_data.xml [hunk 1]
```diff
@@ -14,16 +14,16 @@
     </record>
 
     <record id="account_tax_report_line_post_1" model="account.tax.report.line">
-        <field name="name">Post 1 Samlet omsetning og uttak utenfor merverdiavgiftsloven</field>
-        <field name="tag_name">Post 1 Samlet omsetning og uttak utenfor merverdiavgiftsloven</field>
+        <field name="name">Post 1</field>
+        <field name="tag_name">Post 1</field>
```
Comment (jco-odoo) [inline addons/l10n_no/data/account_tax_report_data.xml]: @nla-odoo We only need to change the tag_name, not the name itself.

Comment (nla-odoo) [reply addons/l10n_no/data/account_tax_report_data.xml]: @jco-odoo  Done

**Comment by jco-odoo** [comment]
@nla-odoo Can you just squash the 2 commits?  (keeping a nice message)  Then we can merge.

**Comment by nla-odoo** [comment]
@jco-odoo  Done

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**